addons/dex/2.22.x/dex-8.yaml

```diff
@@ -0,0 +1,117 @@
+---
+apiVersion: kubeaddons.mesosphere.io/v1beta1
+kind: Addon
+metadata:
+  name: dex
+  namespace: kubeaddons
+  labels:
+    kubeaddons.mesosphere.io/name: dex
+  annotations:
+    catalog.kubeaddons.mesosphere.io/addon-revision: "2.22.0-8"
```

Comment:
The prior revision is `2.22.0-6`. Did you intend to skip 7?

Reply:
I think that was accidentally skipped? the version should generally match the file version.

Reply:
Yeah looks like dex-7 is missing a revision bump. We reset that counter when we bump the patch version, but this is still on `.0` so that obviously hasn't happened. 👍